From a9e737e117ed73bc8dca54e24316b92e9de8e780 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Sun, 29 Mar 2009 11:22:18 +0200
Subject: [PATCH] m68k: Atari EtherNAT - replace bus_id with dev_name()

Cfr. commit db1d7bf70f42124f73675fca62fe32f3ab1111b4 ("net: struct device -
replace bus_id with dev_name(), dev_set_name()") for drivers/net/smc91x.c.

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/net/atari_91C111.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/atari_91C111.c b/drivers/net/atari_91C111.c
index a82ba3d..cac5ede 100644
--- a/drivers/net/atari_91C111.c
+++ b/drivers/net/atari_91C111.c
@@ -1670,7 +1670,7 @@ smc_ethtool_getdrvinfo(struct net_device *dev, struct ethtool_drvinfo *info)
 {
 	strncpy(info->driver, CARDNAME, sizeof(info->driver));
 	strncpy(info->version, version, sizeof(info->version));
-	strncpy(info->bus_info, dev->dev.parent->bus_id, sizeof(info->bus_info));
+	strncpy(info->bus_info, dev_name(dev->dev.parent), sizeof(info->bus_info));
 }
 
 static int smc_ethtool_nwayreset(struct net_device *dev)
-- 
1.5.6.5

